docs/src/main/asciidoc/_project-features-messaging.adoc

```diff
@@ -1298,3 +1301,6 @@ Since the route is set for you, you can send a message to the `{output_name}` de
 include::{tests_path}/spring-cloud-contract-stub-runner-kafka/src/test/groovy/org/springframework/cloud/contract/stubrunner/messaging/kafka/KafkaStubRunnerSpec.groovy[tags=trigger_no_output,indent=0]
 ----
 ====
+
+===== Kafka Record Key
+To
```

Comment:
Sth is missing here I guess

Reply:
I ended up doing the "docs"/explanation in the SCCS rather than here. I forgot to remove this cruft. I missed it on previous commit.